doc-content/enterprise-only/installation/clustering-con.adoc

```diff
@@ -1,7 +1,7 @@
 [id='clustering-con_{context}']
 = {PRODUCT} clusters
 
-By clustering two or more computers, you have the benefits of high availability, enhanced collaboration, and load balancing. High availability decreases the chance of a loss of data when a single computer fails. When a computer fails, another computer fills the gap by providing a copy of the data that was on the failed computer. When the failed computer comes online again, it resumes its place in the cluster. Load balancing shares the computing load across the nodes of the cluster. Doing this improves the overall performance.
+By clustering two or more computers, you have the benefits of high availability, enhanced collaboration, and load balancing. High availability decreases the chance of a loss of data when a single computer fails. When a computer fails, another computer fills the gap by providing a copy of the data that was on the failed computer. When the failed computer comes online again, it resumes its place in the cluster.
```

Comment:
"High availability decreases the chance of a loss of data when a single computer fails."

I suggest a minor change for minimalism. Totally up to you. I could be wrong..

High availability decreases the chance data loss when a single computer fails.

Reply:
Done, will change to "High availability decreases the chance of data loss when a single computer fails."